apps/challenges/urls.py

```diff
@@ -49,6 +49,11 @@
         views.get_challenge_by_pk,
         name="get_challenge_by_pk",
     ),
+    url(
+        r"^challenge/(?P<title>[\w\s]+)/$",
+        views.get_challenge_by_title,
```

Comment:
`get_challenge_by_title `-> `get_challenges_by_title`

Reply:
Refactored variable names in f624c2e